code/modules/surgery/tools.dm

```diff
@@ -261,3 +261,135 @@
 			advanced_surgeries |= OC.advanced_surgeries
 		return TRUE
 	return
+
+/obj/item/scalpel/advanced
+	name = "laser scalpel"
+	desc = "An advanced scalpel which uses laser technology to cut. It's set to scalpel mode."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "scalpel_a"
+	hitsound = 'sound/weapons/blade1.ogg'
+	force = 16
+	toolspeed = 0.25
+	light_color = LIGHT_COLOR_GREEN
+
+/obj/item/scalpel/advanced/Initialize()
+	set_light(1)
+	START_PROCESSING(SSobj, src)
+
+/obj/item/scalpel/advanced/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	return ..()
+
+/obj/item/scalpel/advanced/attack_self(mob/user)
+	playsound(get_turf(user),'sound/machines/click.ogg',50,1)
+	var/obj/item/circular_saw/advanced/saw = new /obj/item/circular_saw/advanced(drop_location())
+	to_chat(user, "<span class='notice'>You incease the power, now it can cut bones.</span>")
+	qdel(src)
+	user.put_in_active_hand(saw)
+
+/obj/item/circular_saw/advanced
+	name = "laser scalpel"
+	desc = "An advanced scalpel which uses laser technology to cut. It's set to saw mode."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "saw_a"
+	hitsound = 'sound/weapons/blade1.ogg'
+	force = 19

```

Comment:
Ayy hol' up a sec.


19 damage sharp weapon that fits in your pocket and doesn't need two hands to use?

Bit beefy innit?

Reply:
a welder does 15 and doesnt require diamonds

Reply:
It's also not sharp, less than 19, does not cause bleeding and runs out of fuel.